README.md

```diff
@@ -112,3 +112,13 @@ The master repository for the tech docs template system is at https://github.com
 See the section "Updating a project to use the latest template" at https://github.com/alphagov/tech-docs-template
 
 The template system is maintained and developed by the service design manual team.  You can contact them at `#tech-docs-format` on GDS Slack. Note that this is an MVP version and there is a roadmap of planned features and improvements.
+
+## Licence
+
+Unless stated otherwise, the codebase is released under [the MIT licence](./LICENSE).
+
+The data is [© Crown
```

Comment:
I noticed that the Google Doc says "documentation" but we say "data", presumably because we're talking about the non-code files in this repo and not documentation about how to use this repo?

Reply:
Yes, although this might need to be clarified going forward (it's under discussion within tech writer community). If I need to amend it I will.